### ISSUE TITLE
Hide "Settings" tab by default

### ISSUE THREAD

**Issue by cowwoc** [oracle/visualvm#121]
Please add a configuration option that allows us to hide the Sampler/Profiler settings tab by default. This is useful if we are repeatedly profiling an application, terminating it and repeating. Having to repeatedly hide the settings gets annoying.

**Comment by jisedlac** [comment]
Sampler settings are hidden by default. The 'Default' settings for Profiler must be initially shown as there is no common configuration for Memory.

We could hide the Settings view by default if there's a custom configuration available for the process with all the required settings defined. That should be intuitive enough.

**Comment by cowwoc** [comment]
> Sampler settings are hidden by default. The 'Default' settings for Profiler must be initially shown as there is no common configuration for Memory.

You are right. I am only doing profiling (not sampling). Also, I am only using CPU profiling, not memory profiling.

I believe that you *could* hide the settings for the profiler by default because if a user clicks on the Memory button they get an error message "Provided Memory settings are invalid" and the settings dialog pops open.
